nmigen_boards/mister.py

```diff
@@ -65,44 +65,40 @@ class MisterPlatform(IntelPlatform):
+            dqm="", conn=("gpio_0", 1), attrs=Attrs(io_standard="3.3-V LVCMOS")),
 
         # MiSTer I/O Board (optional, but highly recommended)
         # https://github.com/MiSTer-devel/Hardware_MiSTer/blob/master/releases/iobrd_6.0.pdf
-        Resource("power_led", 0, PinsN("1", dir="o", conn=("gpio", 1)), Attrs(io_standard="3.3-V LVTTL")),
-        Resource("disk_led", PinsN("3", dir="o", conn=("gpio", 1)), Attrs(io_standard="3.3-V LVTTL")),
-        Resource("user_led", PinsN("5", dir="o", conn=("gpio", 1)), Attrs(io_standard="3.3-V LVTTL")),
+        Resource("power_led", 0, PinsN("1", dir="o", conn=("gpio_1", 1)), Attrs(io_standard="3.3-V LVTTL")),
+        Resource("disk_led", PinsN("3", dir="o", conn=("gpio_1", 1)), Attrs(io_standard="3.3-V LVTTL")),
+        Resource("user_led", PinsN("5", dir="o", conn=("gpio_1", 1)), Attrs(io_standard="3.3-V LVTTL")),
 
-        Resource("reset_switch", 0, PinsN("17", dir="i", conn=("gpio", 1)), Attrs(io_standard="3.3-V LVTTL")),
-        Resource("osd_switch", 0, PinsN("13", dir="i", conn=("gpio", 1)), Attrs(io_standard="3.3-V LVTTL")),
-        Resource("user_switch", 0, PinsN("15", dir="i", conn=("gpio", 1)), Attrs(io_standard="3.3-V LVTTL")),
+        Resource("reset_switch", 0, PinsN("17", dir="i", conn=("gpio_1", 1)), Attrs(io_standard="3.3-V LVTTL")),
+        Resource("osd_switch", 0, PinsN("13", dir="i", conn=("gpio_1", 1)), Attrs(io_standard="3.3-V LVTTL")),
+        Resource("user_switch", 0, PinsN("15", dir="i", conn=("gpio_1", 1)), Attrs(io_standard="3.3-V LVTTL")),
 
         Resource("audio", 0,
-            Subsignal("l", Pins("2", dir="o", conn=("gpio", 1))),
-            Subsignal("r", Pins("7", dir="o", conn=("gpio", 1))),
+            Subsignal("l", Pins("2", dir="o", conn=("gpio_1", 1))),
+            Subsignal("r", Pins("7", dir="o", conn=("gpio_1", 1))),
             Attrs(io_standard="3.3-V LVTTL")),
 
-        Resource("toslink", 0, Pins("9", dir="o", conn=("gpio", 1))),
+        Resource("toslink", 0, Pins("9", dir="o", conn=("gpio_1", 1))),
 
         *SDCardResources(0,
-            clk="gpio_1:13", cmd="gpio_1:6", 
-            dat0="gpio_1:16", dat1="gpio_1:18", dat2="gpio_1:4", dat3="gpio_1:6",
-            attrs=Attrs(io_standard="3.3-V LVTTL")),
+            clk="13", cmd="6", 
+            dat0="16", dat1="18", dat2="4", dat3="6",
+            conn=("gpio_1", 1), attrs=Attrs(io_standard="3.3-V LVTTL")),
 
         # The schematic is difficult to understand here...
         Resource("vga", 0,
-            Subsignal("r", Pins("28 32 34 36 38 40", dir="o", conn=("gpio", 1))),
-            Subsignal("g", Pins("27 31 33 35 37 39", dir="o", conn=("gpio", 1))),
-            Subsignal("b", Pins("21 23 25 26 24 24", dir="o", conn=("gpio", 1))),
-            Subsignal("hs", Pins("20", dir="o", conn=("gpio", 1))),
-            Subsignal("vs", Pins("19", dir="o", conn=("gpio", 1))),
+            Subsignal("r", Pins("28 32 34 36 38 40", dir="o", conn=("gpio_1", 1))),
```

Comment:
Shouldn't this be `conn=("gpio", 1)`?

Reply:
Ah, I got confused with your other comment. I'll probably fix this tomorrow.